BUILD.md

```diff
@@ -171,7 +171,8 @@ tables/table-rhel7-stig.html
 ### Testing
 
 To ensure validity of built artifacts prior to installation, we recommend
-running our test suite against the build output. This is done with CTest:
+running our test suite against the build output. This is done with CTest
+and you need to also install `yamllint` and `ansible-lint` packages:
```

Comment:
I have noticed that there is an optional test dependency ShellChekc. Is it possible to describe installing both packages at the same place in the docs?

Reply:
Done. I moved the part about packages to the proper place.